### PR TITLE
Add Norms (again)

### PR DESCRIPTION
**Pull request by NickHeiner**
Like #3009, but hopefully this time Travis doesn't do the weird thing where it fails to load the s3 creds.

### CODE REVIEW

File: docs/misc_eng_norms.md [hunk 1]
```diff
@@ -67,3 +89,9 @@ When in doubt, overcommunicate.
 ## Pair Programming
  
 Pair programming is the one of the quickest ways to expand your knowledge of the codebase. When you learn of an interesting ticket during standup, ask that engineer if you can pair with them on it. Similarly, engineers working on a particularly challenging ticket should ask for someone to pair with them. A second set of eyes during development can improve the quality of the code and reduce bugs. 
+
+## Be Mindful of Ops Impact of New Features
+When you roll out a new feature, be mindful of the impact it'll have on our larger system, and on the VA as a whole. For instance, if you change the way that Caseflow talks to VBMS, you should monitor our VBMS traffic to ensure that we aren't putting excessive new load on that system.
+
+## Verifying Code on UAT
+If you have code that talks to external dependencies, push your PR branch to UAT to validate it. This will provide a much better test than trying to run locally with dependencies mocked out. Do not rely on QA's integration tests to catch errors.
```
Comment (kierachell) [inline docs/misc_eng_norms.md]: Do not rely on integration tests to catch edge case errors: they are smoke tests - not complete regression/feature tests

Comment (NickHeiner) [inline docs/misc_eng_norms.md]: Sounds like a good update – want to send a PR for it?
